```
18:05:42  Traceback (most recent call last):
18:05:42    File "C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\swift\utils\line-directive", line 738, in <module>
18:05:42      run()
18:05:42    File "C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\swift\utils\line-directive", line 689, in run
18:05:42      command = subprocess.Popen(
18:05:42    File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\lib\subprocess.py", line 951, in __init__
18:05:42      self._execute_child(args, executable, preexec_fn, close_fds,
18:05:42    File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\lib\subprocess.py", line 1420, in _execute_child
18:05:42      hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
18:05:42  FileNotFoundError: [WinError 206] The filename or extension is too long
18:05:42  Batch file failed at line 6 with errorcode 1
```

--- a/lldb/cmake/modules/LLDBConfig.cmake
+++ b/lldb/cmake/modules/LLDBConfig.cmake
@@ -51,6 +51,11 @@ macro(add_optional_dependency variable description package found)
   message(STATUS "${description}: ${${variable}}")
 endmacro()
 
+set(LLDB_LIBXML2_VERSION "2.8" CACHE STRING
+  "Specify the version of libxml 2 to use with LLDB. This is only meant to be overridden for local
+  static builds of libxml 2. Use at your own risk.")
+mark_as_advanced(LLDB_LIBXML2_VERSION)
+
 add_optional_dependency(LLDB_ENABLE_SWIG "Enable SWIG to generate LLDB bindings" SWIG SWIG_FOUND VERSION 4)
 
 # BEGIN SWIFT MOD
@@ -67,7 +72,7 @@ add_optional_dependency(LLDB_ENABLE_CURSES "Enable curses support in LLDB" Curse
 add_optional_dependency(LLDB_ENABLE_LZMA "Enable LZMA compression support in LLDB" LibLZMA LIBLZMA_FOUND)
 add_optional_dependency(LLDB_ENABLE_LUA "Enable Lua scripting support in LLDB" LuaAndSwig LUAANDSWIG_FOUND)
 add_optional_dependency(LLDB_ENABLE_PYTHON "Enable Python scripting support in LLDB" PythonAndSwig PYTHONANDSWIG_FOUND)
-add_optional_dependency(LLDB_ENABLE_LIBXML2 "Enable Libxml 2 support in LLDB" LibXml2 LIBXML2_FOUND VERSION 2.8)
+add_optional_dependency(LLDB_ENABLE_LIBXML2 "Enable Libxml 2 support in LLDB" LibXml2 LIBXML2_FOUND VERSION ${LLDB_LIBXML2_VERSION})
 add_optional_dependency(LLDB_ENABLE_FBSDVMCORE "Enable libfbsdvmcore support in LLDB" FBSDVMCore FBSDVMCore_FOUND QUIET)
 
 option(LLDB_USE_ENTITLEMENTS "When codesigning, use entitlements if available" ON)

